### PR TITLE
Update Customer Churn ZenFile to ZenML 0.20.2

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Update the Custom Churn ZenFile to work with ZenML 0.20.2!

## Pre-requisites
Please ensure you have done the following:
- [X] If my change requires a change to docs, I have updated the documentation accordingly.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Other (add details above)

### CODE REVIEW

File: customer-churn/README.md [hunk 1]
```diff
@@ -60,40 +60,12 @@ Before going on next step, let's review some of the core concepts of ZenML:
 - **Container registry**: Some orchestrators will require you to containerize the steps of your pipeline. A container registry is a store for these (Docker) containers. A ZenML workflow involving a container registry will containerize your code and store the resulting container in the registry.
 - **Kubeflow orchestrator**: An orchestrator manages the running of each step of the pipeline, administering the actual pipeline runs. It controls how and where each step within a pipeline is executed.
 
-### Run the pipeline on a local Kubeflow Pipelines deployment
+### Run the pipeline on a local orchestrator Pipelines deployment
```
Comment (AlexejPenner) [inline customer-churn/README.md:63]:
```suggestion
### Run the pipeline on a local orchestrator
```

File: customer-churn/README.md [hunk 2]
```diff
@@ -108,80 +80,79 @@ To run the model training pipeline:
...
-export ECR_REGISTRY_NAME=<your ECR registry name>
-# the S3 bucket name
-export S3_BUCKET_NAME=<your S3 bucket name>
-# the namespace where Kubeflow is installed in the remote cluster
-export KUBEFLOW_NAMESPACE=kubeflow
-```
+    ```shell
+    zenml stack recipe pull aws_minimal
+    ```
+2. (Optional) 🎨 Customize your deployment by editing the default values in the `locals.tf` file.
```
Comment (AlexejPenner) [inline customer-churn/README.md:107]: You could write one sentece explaining what this does

**Review by AlexejPenner — CHANGES_REQUESTED** [review]
Two small improverments, otherwise looks good

File: customer-churn/README.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ pip install -r requirements.txt
 We also need to install the following ZenML integrations for this project:
 
 ```bash
-zenml integration install -y facets sklearn xgboost lightgbm kubeflow
+zenml integration install -y facets sklearn xgboost lightgbm kubeflow seldon
```
Comment (htahir1) [inline customer-churn/README.md:29]: -y at the end

File: customer-churn/README.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ We showcase two solutions to this problem:
 - `Deployment using Kubeflow Pipelines`: We will be using [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/) to build and run our ZenML pipeline on the cloud and deploy it in a production environment.
 - `Continuous Deployment using Seldon Core`: We will be using [Seldon Core](https://docs.seldon.io/projects/seldon-core/en/latest/index.html), a production-grade open-source model serving platform, to build our continuous deployment pipeline that trains a model and then serves it with Seldon Core.
 
-## Deploy pipelines to production using Kubeflow Pipelines
+## Deploy pipelines to production using orchestrator Pipelines
```
Comment (htahir1) [inline customer-churn/README.md:43]: not sure if this is the right change?

File: customer-churn/README.md [hunk 5]
```diff
@@ -108,80 +80,80 @@ To run the model training pipeline:
...
-
-# Register the cloud stack
-zenml stack register cloud_kubeflow_stack \
-  -m kubeflow_metadata_store \
-  -a cloud_artifact_store \
-  -o cloud_orchestrator \
-  -c cloud_registry
-```
+    ```bash
+    zenml secret register aws_rds_secret \
```
Comment (htahir1) [inline customer-churn/README.md:133]: Do we need the rds secret now? I dont think so

**Review by htahir1 — CHANGES_REQUESTED** [review]
Left a quick review

**Pull request merged**